regression/test-container.sh

```
@@ -0,0 +1,33 @@
+
+# Must ensure PROJECT_DIR is exported first then load rapids-mg-tools env
+export PROJECT_DIR=${PROJECT_DIR:-$(cd $(dirname ${BASH_SOURCE[0]}) && pwd)}
+source ${PROJECT_DIR}/config.sh
+
+################################################################################
+
+# Test
+logger "Testing container image $IMAGE"
+python -c "import cuopt; print(cuopt)"
+
+trap "${SCRIPTS_DIR}/write-meta-data.sh" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)